packages/commercetools/composables/src/helpers/internals/enhanceProduct.ts

```diff
@@ -38,7 +38,7 @@ const enhanceProduct = (productResponse: ApolloQueryResult<ProductData>, context
         _description: current.description,
         _categoriesRef: current.categoriesRef.map((cr) => cr.id),
         _rating: (product as any).reviewRatingStatistics,
-        ...(product.productType && { _productType: product.productType })
+        _original: product
```

Comment:
`productType` is no longer needed?

Reply:
@dawid-ziobro not really, since from now on any custom field (like productType) can be found in the _original obj
<img width="536" alt="Screenshot 2021-10-01 at 17 47 24" src="https://user-images.githubusercontent.com/34214733/135649976-30da0fa2-9736-41f0-8111-aa6385546d1b.png">
.